perf/Grpc.AspNetCore.Microbenchmarks/Internal/MessageHelpers.cs

```diff
@@ -21,20 +21,22 @@
 using Google.Protobuf;
 using Grpc.AspNetCore.Server;
 using Grpc.AspNetCore.Server.Internal;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging.Abstractions;
 
 namespace Grpc.AspNetCore.Microbenchmarks.Internal
 {
     internal static class MessageHelpers
     {
-        private static readonly GrpcServiceOptions TestServiceOptions = new GrpcServiceOptions();
+        private static readonly HttpContextServerCallContext TestServerCallContext = new HttpContextServerCallContext(new DefaultHttpContext(), new GrpcServiceOptions(), NullLogger.Instance);
 
         public static void WriteMessage<T>(Stream stream, T message) where T : IMessage
         {
             var messageData = message.ToByteArray();
 
             var pipeWriter = new StreamPipeWriter(stream);
 
-            PipeExtensions.WriteMessageAsync(pipeWriter, messageData, TestServiceOptions, flush: true).GetAwaiter().GetResult();
+            PipeExtensions.WriteMessageAsync(pipeWriter, messageData, TestServerCallContext, flush: true).GetAwaiter().GetResult();
```

Comment:
I had an initial approach that needed the context which is why the pipe extensions now take it, but it isn't required right now. I suspect at some point we'll need info off it so I have left it this way for now.